doc/nrf/libraries/nrf_security/doc/driver_config.rst

```diff
@@ -506,14 +552,63 @@ To enable password-authenticated key agreement support, configure the related pa
++-----------------------+--------------------------+---------------------------+
+| SRP-6                 | Not supported            | Supported                 |
++-----------------------+--------------------------+---------------------------+
+
+Configuration of the :ref:`nrf_security_drivers_oberon` driver is automatically generated based on the user-enabled algorithms in  `Password-authenticated key agreement configurations`_.
+
+Key size configurations
+***********************
+
+:ref:`nrf_security` supports key size configuration options for AES and RSA keys.
+
+
+AES key size configuration
+==========================
+
++--------------+----------------------------------------------------+
+| AES key size | Configuration option                               |
++==============+====================================================+
+| 128 bits     | :kconfig:option:`CONFIG_PSA_WANT_AES_KEY_SIZE_128` |
++--------------+----------------------------------------------------+
+| 192 bits     | :kconfig:option:`CONFIG_PSA_WANT_AES_KEY_SIZE_192` |
++--------------+----------------------------------------------------+
+| 256 bits     | :kconfig:option:`CONFIG_PSA_WANT_AES_KEY_SIZE_256` |
++--------------+----------------------------------------------------+
+
+.. note::
+   * All AES key size configurations are introduced by :ref:`nrf_security` and are not described by the PSA Crypto specification.
+
+
+RSA key size configuration
+==========================
+
++--------------------+-----------------------------------------------------+
+| RSA key size       | Configuration option                                |
++====================+=====================================================+
+| 1024 bits          | :kconfig:option:`CONFIG_PSA_WANT_RSA_KEY_SIZE_1024` |
++--------------------+-----------------------------------------------------+
+| 1536 bits          | :kconfig:option:`CONFIG_PSA_WANT_RSA_KEY_SIZE_1536` |
++--------------------+-----------------------------------------------------+
+| 2048 bits          | :kconfig:option:`CONFIG_PSA_WANT_RSA_KEY_SIZE_2048` |
++--------------------+-----------------------------------------------------+
+| 3072 bits          | :kconfig:option:`CONFIG_PSA_WANT_RSA_KEY_SIZE_3072` |
++--------------------+-----------------------------------------------------+
+| 4096 bits          | :kconfig:option:`CONFIG_PSA_WANT_RSA_KEY_SIZE_4096` |
++--------------------+-----------------------------------------------------+
+| 6144 bits          | :kconfig:option:`CONFIG_PSA_WANT_RSA_KEY_SIZE_6144` |
++--------------------+-----------------------------------------------------+
+| 8192 bits          | :kconfig:option:`CONFIG_PSA_WANT_RSA_KEY_SIZE_8192` |
++--------------------+-----------------------------------------------------+
+
+.. note::
+   * All RSA key size configurations are introduced by :ref:`nrf_security` and are not described by the PSA Crypto specification.
```

Comment:
I don't remember if it was in reference to this or another PR, but I checked this formatting with Bartek. He said that notes should have bullet points only if there are multiple items in the list.

Please remove bullet points for notes with only one item.